deployment/ansible/roles/model-my-watershed.tiler/tasks/configuration.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Configure Node Modules mount
```

Comment:
Should these two items be guarded in an `if ['development, 'test']` block?

Reply:
Yes:

```yaml
when: "['development', 'test'] | some_are_in(group_names)"
```

We only need this to apply within Vagrant based environments where synced folders are in play.

Reply:
It seems like this should only be a dev/test thing, just like the `mound --bind` call it is replacing.

Reply:
Updated to be so in b9fe207.